src/NIST-800-53/nist-800-53.ts

```diff
@@ -508,6 +516,19 @@ export class NIST80053Checks extends NagPack {
    * @param ignores list of ignores for the resource
    */
   private checkRDS(node: CfnResource, ignores: any): void {
+    if (
+      !this.ignoreRule(ignores, 'NIST.800.53-DBInstanceBackupEnabled') &&
```

Comment:
For clarity purposes what do you think about renaming the Rule to  `NIST.800.53-RDSInstanceBackupEnabled` considering it only applies to RDS. It would also make looking up RDS rules on the rules list a lot easier